service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/docmosis/OrderCreationServiceDocmosisTest.java

```diff
@@ -111,6 +116,7 @@ private String getExpectedText(Order order) {
 
     private static Stream<Arguments> allOrders() {
         return Stream.of(Order.values())
+            .filter(order -> order.getSourceType() == OrderSourceType.DIGITAL)
```

Comment:
vv minor but would it be nice to have an `order.isDigital` method? if you do want it then remember to annotate the method with `@JsonIgnore`